goosebit/settings/schema.py

```diff
@@ -63,7 +63,7 @@ def settings_customise_sources(
     ) -> tuple[PydanticBaseSettingsSource, ...]:
         return (
             init_settings,
-            YamlConfigSettingsSource(settings_cls, BASE_DIR.parent.joinpath("settings.yaml")),
+            YamlConfigSettingsSource(settings_cls, BASE_DIR.joinpath("settings.yaml")),
```

Comment:
We have to keep in mind that we are creating a PyPI package. With that installed, this will point to e.g. `/usr/local/lib/python3.12/site-packages`, which is definitely not a good location. Anyway, this was already broken before.

Reply:
I think instead of building this path ourselves, we should look into how to do this with a library.

Reply:
Could also just allow the user to pass it as an arg to `main.py`, simple enough?

Reply:
Created issue #103